docs/ossm/README.md

```diff
@@ -6,4 +6,80 @@ This documentation is specific to the OpenShift Service Mesh product and may dif
 
 # Table of Contents
 
-TODO
+- [Running OSSM 2 and OSSM 3 side by side](#running-ossm-2-and-ossm-3-side-by-side)
+
+
+# Running OSSM 2 and OSSM 3 side by side
+This section describes how to run OSSM 2 and OSSM 3 side by side in one cluster without interfering with each other.
+> **_NOTE:_** It's not OSSM 2 -> OSSM 3 migration guide.
+
+To better understand steps described here, it's recommended to read OSSM 2 [deployment models](https://docs.redhat.com/en/documentation/openshift_container_platform/4.17/html/service_mesh/service-mesh-2-x#ossm-deployment-models) and upstream Istio [Multiple Control Planes](https://istio.io/latest/docs/setup/install/multiple-controlplanes/) documentation. This document assumes OSSM 2.6 is already running and the goal is to install OSSM 3.0.
+
+Depending on your OSSM 2 model, follow steps described in next chapters.
+
+## Cluster-Wide (Single Tenant) mesh deployment model
+### Required OSSM 2 changes
```

Comment:
Maybe add "...when running in cluster-wide mode" to the title to make it clearer that it is only for cluster wide mode.